src/index.ts

```diff
@@ -1,20 +1,35 @@
 import mongodb from 'mongodb';
-import { menash } from 'menashmq';
-import { ConnectionStringParser as CSParser, IConnectionStringParameters } from 'connection-string-parser';
-import { MongoDataType, RabbitDataType, DataObjectType, MTROptions } from './paramTypes';
 import log from './utils/logger';
+import { MongoDataType,
+         RabbitDataType,
+         DataObjectType,
+         MTROptions,
+         MiddlewareFuncType, 
+         QueueObjectType
+        } from './paramTypes';
+import { ConnectionStringParser as CSParser,
+         IConnectionStringParameters } from 'connection-string-parser';
+import {menash} from 'menashmq';
+import { ChangeStreamOptions } from 'mongodb';
+import { MongoClient } from 'mongodb';
 
+let mongoConn: MongoClient;
 const csParser = new CSParser({
     scheme: 'mongodb',
     hosts: []
 });
 
+const defaultMiddleware: MiddlewareFuncType = (data: DataObjectType) => {
+    return {data};
```

Comment:
Why does it encapsulate the data inside of another object?

Reply:
My bad, it was an object at first and I forgot to delete the curly brackets, nice catch!